src/javascript/app_2/Documents/STYLE_GUIDE.MD

```diff
@@ -0,0 +1,249 @@
+    │   │   ├── Containers/
+    │   │   │   ├── trade_form.jsx
+    │   │   │   └── trade.jsx
+    │   │   ├── Helpers/
+    │   │   └── index.js
+    │   └── Statement/
+    │       ├── Components/
+    │       ├── Containers/
+    │       ├── Helpers/
+    │       └── index.js
+    ├── Services/
+    ├── Stores/
+    ├── Utils/
+    │   ├── Date/
+    │   └── …
+    ├── Documents/
+    └── index.js
+```
+
+This guide is based on presentational and container components. 
+If you're not sure of the distinction between them, we recommend you read the 
+[Presentational and Container Components](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0) 
+article first.
+
+## App
+
+The `App` directory contains common components and containers of the application
+which are used in others modules (e.g. `Form`, `Layout`).
+
+## Constants
+
+All public and common constants variable should be stored in related files in this directory
+(e.g. contract_type_display).
+
+## Documents
+
+This directory contains all documentations of the project (e.g. STYLE_GUIDE.RM).
+
+## MODULES
+
+The predominant part of the project is the `Modules` directory. 
+All the presentational and container components are classified by functionality 
+here. Regarding the picture above, each functionality in the project has a 
+directory in `Modules` directory (e.g. `Trading`) that contains two major subdirectories 
+`Components` and `Containers`. We will explain more about `Components` and 
+`Containers` directories in further sections.
+
+
+### COMPONENTS
+
+This directory contains React components, of either stateless or stateful varieties 
+(e.g. `function` or `class` components).
+
+The optimal structure for components is to place files relevant to a component inside 
+its own subdirectory, e.g.:
+
+```
+Components/
+├── Elements/
+│   ├── full_screen_dialog.jsx
+│   └── mobile_widget.jsx
+└── Form/
+    ├── ContractType/
+    ├── Purchase/
+    ├── TradeParams/
+    ├── form_layout.jsx
+    ├── screen_large.jsx
+    └── screen_small.jsx
+```
+
+The presentational components:
+
+* Are concerned with how things look.
+* May contain both presentational and container components inside, 
+and usually, have some DOM markup and styles of their own.
+* Often allow containment via this.props.children.
+* Have no dependencies on the rest of the app, such as Flux actions or stores.
+* Don’t specify how the data is loaded or mutated.
+* Receive data and callbacks exclusively via props.
+* Rarely have their own state (when they do, it’s UI state rather than data).
+* Are written as functional components unless they need state, lifecycle hooks, 
+or performance optimizations.
+* Examples: Balance, AccountList, UserIfo, ...
+
+#### Do:
+
+* Keep all files immediately relevant to a component inside the given component directory.
+
+* Add a basic `index.js` to import the component and export it 
+to make it dead simple to import the component from elsewhere.
```

Comment:
maybe other wording instead of `dead simple`? 😅

Reply:
I like the use of `dead simple` though 😄

Reply:
I vote for dead simple 🗡

Reply:
@negar-binary 
What is your suggestion?

Reply:
keep it if it's popular :)